__tests__/add-to-project.test.ts

```diff
@@ -90,48 +90,96 @@ describe('addToProject', () => {
         return: {
           organization: {
             projectV2: {
-              id: 'project-id'
-            }
-          }
-        }
+              id: 'project-id',
+            },
+          },
+        },
       },
       {
         test: /addProjectV2DraftIssue/,
         return: {
           addProjectV2DraftIssue: {
             projectItem: {
-              id: 'project-item-id'
-            }
-          }
-        }
-      }
+              id: 'project-item-id',
+            },
+          },
+        },
+      },
     )
 
     await addToProject()
 
     expect(outputs.itemId).toEqual('project-item-id')
   })
 
+  test('add an issue with fields product and client', async () => {
+    mockGetInput({
+      'project-url': 'https://github.com/orgs/actions/projects/1',
+      'github-token': 'gh_token',
+      fields: 'product=test-product, client=test-client',
+    })
+
+    github.context.payload = {
+      issue: {
+        number: 1,
+        labels: [{name: 'bug'}],
+        // eslint-disable-next-line camelcase
+        html_url: 'https://github.com/actions/add-to-project/issues/74',
+      },
+      repository: {
+        name: 'add-to-project',
+        owner: {
+          login: 'actions',
+        },
+      },
+    }
+
+    mockGraphQL(
+      {
+        test: /getProject/,
+        return: {
+          organization: {
+            projectV2: {
+              id: 'project-id',
+            },
+          },
+        },
+      },
+      {
+        test: /addProjectV2ItemById/,
+        return: {
+          addProjectV2ItemById: {
+            item: {
+              id: 'project-item-id',
+            },
+          },
+        },
+      },
+    )
+    await addToProject()
+    expect(outputs.itemId).toEqual('project-item-id')
```

Comment:
Can we check here that the fields were added correctly? Does the mock allows us to verify a specific call was made?